tests/test_image_types.py

```diff
@@ -0,0 +1,337 @@
+import shutil
+
+import wagtail_factories
+
+from django.conf import settings
+from django.test import override_settings
+from test_grapple import BaseGrappleTest, BaseGrappleTestWithIntrospection
+from wagtail import VERSION as WAGTAIL_VERSION
+from wagtail.images import get_image_model
+
+from grapple.types.images import rendition_allowed
+
+
+class ImageTypesTest(BaseGrappleTestWithIntrospection):
+    @classmethod
+    def setUpTestData(cls):
+        cls.image_model = get_image_model()
+        cls.example_image = wagtail_factories.ImageFactory(title="Example Image")
+        cls.example_image.full_clean()
+        cls.example_image.save()
+
+    @classmethod
+    def tearDownClass(cls):
+        super().tearDownClass()
+        shutil.rmtree(settings.MEDIA_ROOT, ignore_errors=True)
```

Comment:
**Suggestion:** Add a comment to clarify that this media directory only exists within the `tests` dir and we're not removing the project's main `media` directory.

Reply:
reworked this entirely